src/components/GanttChart.vue

```diff
@@ -122,3 +114,8 @@ export default {
   },
 };
 </script>
+<style>
+.g-timeaxis {
```

Comment:
use correct spelling

Reply:
![image](https://user-images.githubusercontent.com/99862978/225607266-01f68673-c6c1-4101-b325-2f3984e6a807.png)
It's a default css class